src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
+        int index = random.nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
```

Comment:
Use `name()` instead of `toString()` for getting the string representation of enum constants to avoid issues if `toString()` is overridden in the enum. Replace `toString()` with `name()`.

Reply:
fix comment